### PR TITLE
Errors/1162 fix

### PR DESCRIPTION
**Pull request by ChopperTheKing**
The develop branch shouldn't have any errors now. But the code is still wonky af lol. If anyone wants to take it from here, be my guest.

### CODE REVIEW

File: DeBlockWallet.xcodeproj/xcshareddata/xcschemes/DeBlockWallet.xcscheme
Comment (mattmaloof) [inline DeBlockWallet.xcodeproj/xcshareddata/xcschemes/DeBlockWallet.xcscheme:1]: is this file needed?

**Review by k-code00 — APPROVED** [review]
good job fixing the errors. will pull right away

**Review by mattmaloof — APPROVED** [review]
looks fine.

File: DeBlockWallet.xcodeproj/xcshareddata/xcschemes/DeBlockWallet.xcscheme
Comment (ChopperTheKing) [reply DeBlockWallet.xcodeproj/xcshareddata/xcschemes/DeBlockWallet.xcscheme:1]: Looks like a junk file. Remove if you want

Comment (ChopperTheKing) [reply DeBlockWallet.xcodeproj/xcshareddata/xcschemes/DeBlockWallet.xcscheme:1]: Actually nah. CGPT says that that type of file is used to store configuration data for the project
